### PR TITLE
Add NodeEdgeLabelMapping

### PR DESCRIPTION
**Pull request by BentolhodaH**
Add NodeEdgeLabelMapping
Add test for NodeEdgeLabelMapping

### CODE REVIEW

File: src/main/java/se/liu/ida/hefquin/vocabulary/LPG2RDF.java [hunk 1]
```diff
@@ -24,10 +24,13 @@ protected static final Property property(final String local ) {
     public static final Resource NodeLabelMapping = resource("NodeLabelMapping");
     public static final Resource IRIBasedNodeLabelMapping = resource("IRIBasedNodeLabelMapping");
     public static final Resource LiteralBasedNodeLabelMapping = resource("LiteralBasedNodeLabelMapping");
+    public static final Resource NodeEdgeLabelMapping = resource("NodeEdgeLabelMapping");
+    public static final Resource IRIBasedNodeEdgeLabelMapping = resource("IRIBasedNodeEdgeLabelMapping");
 
     public static final Property labelPredicate = property("labelPredicate");
     public static final Property nodeMapping = property("nodeMapping");
     public static final Property nodeLabelMapping = property("nodeLabelMapping");
+    public static final Property nodeEdgeLabelMapping = property("nodeEdgeLabelMapping");
```
Comment (hartig) [inline src/main/java/se/liu/ida/hefquin/vocabulary/LPG2RDF.java:33]:
```suggestion
    public static final Property edgeLabelMapping = property("edgeLabelMapping");
```

**Review by hartig — CHANGES_REQUESTED** [review]
That was fast! Great! Thanks for your efforts!!

However, there is an issue with the names of the new RDF classes and property and, by extension, also with the Java class names. Yet, these things are easy to fix.

File: ExampleLPG2RDF.ttl [hunk 2]
```diff
@@ -25,6 +25,11 @@ PREFIX ex:     <http://example.org/>
 #   rdfs:subClassOf ex:NodeLabelMapping .
 #
 
+#ex:NodeEdgeLabelMapping a rdfs:Class ;
+#
+#ex:IRIBasedNodeEdgeLabelMapping a rdfs:Class ;
+#   rdfs:subClassOf ex:NodeEdgeLabelMapping .
```
[the remark this answers is not in the document]
Comment (hartig) [reply ExampleLPG2RDF.ttl:31]: @BentolhodaH 
You forgot to change these lines. If you adapt them as well, then this PR is ready to be merged.

[the remark this answers is not in the document]
Comment (BentolhodaH) [reply ExampleLPG2RDF.ttl]: I applied them.

**Review by hartig — APPROVED** [review]

**Pull request merged**